src/docs/development/tools/devtools/code-size.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Using the code size tool
+description: Learn how to use the DevTools code size tool.
+---
+
+## What is it?
+
+The code size tool allows you to analyze the total size of your app, including
+both native and Dart code, or just the size of the Dart code in your app. You
+can view a single snapshot of "size information" using the [Analysis tab][], or
+compare two different snapshots of "size information" using the [Diff tab][].
+
+### What is "size information"?
+
+The Dart AOT compiler performs tree-shaking on your code when compiling your
+application. This means that the compiler attempts to optimize your app's size
+by removing pieces of code that are unused or unreachable.
+
+After the compiler optimizes your code as much as it can, the end result can be
+summarized as the collection of packages, libraries, classes, and functions that
+exist in the binary output, along with their size in bytes. This is the "size
+information" we can analyze in the code size tool to further optimize code and
+track down size issues.
+
+## How to use it
+
+If DevTools is already connected to a running application, navigate to the
+"Code Size" tab.
+
+![Screenshot of code size tab]({% asset tools/devtools/code_size_tab.png @path %})
+
+If DevTools is not connected to a running application, you can
+access the tool from the landing page that appears once you have launched
+DevTools (see [installation instructions][]).
```

Comment:
May or may not be an existing issue. It works on Chrome but I wasn't able to get it to work on Safari. I don't think we mention explicitly in https://flutter.dev/docs/development/tools/devtools/overview that it works on Chrome only. Maybe I'm holding it wrong too :)

Reply:
I believe we say somewhere that we only explicitly support chrome...I could be wrong though - maybe we've only stated this on github issues

Reply:
Looking at the sub pages in https://flutter.dev/docs/development/tools/devtools, I think it's not said very explicitly.